src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -91,6 +91,7 @@ void ChordLayout::layout(Chord* item, LayoutContext& ctx)
 
 void ChordLayout::layoutPitched(Chord* item, LayoutContext& ctx)
 {
+    PaddingTable paddingTable = item->score()->paddingTable();
```

Comment:
1. Please declare this closer to the place where it's used
2. It should be **const** PaddingTable**&** paddingTable